AnnotatoBackend/Sources/App/routes.swift

```diff
@@ -8,4 +9,13 @@ func routes(_ app: Application) throws {
     app.get("hello") { _ -> String in
         "Hello, world!"
     }
+
+    // TODO: Remove after testing: Creates a document and returns the id of the created document as the response
+    app.get("test_documents") { _ -> String in
+        let testDocument = Document(name: "Test Document", ownerId: UUID(), baseFileUrl: "path/to/document")
+        let testDocumentEntity = DocumentEntity.fromModel(testDocument)
+        testDocumentEntity.save(on: app.db)
+
+        return String(describing: testDocumentEntity.id)
+    }
```

Comment:
I am not sure if I am doing it correctly but making a get request to `test_documents` shows me this

<img width="396" alt="Screenshot 2022-03-13 at 10 05 52 PM" src="https://user-images.githubusercontent.com/53614454/158063343-bdc6bae4-4dd2-4472-b23b-2be4e8c1ab8c.png">
 
but does not seem to create the db entry.